Android/app/src/main/java/com/example/samplestickerapp/ContentFileParser.java

```diff
@@ -165,7 +169,9 @@ private static StickerPack readStickerPack(@NonNull JsonReader reader) throws IO
                         }
                     }
                     reader.endArray();
-                } else {
+
+                }
+                else {
```

Comment:
revert？

Reply:
yeah, this one is solved in the last commit, reformat.